pkg/types/aws/validation/machinepool.go

```diff
@@ -7,9 +7,25 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
+	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/aws"
 )
 
+var (
+	validArchitecturesAWS = map[types.Architecture]bool{
+		types.ArchitectureAMD64: true,
+		types.ArchitectureARM64: true,
+	}
+
+	validArchitectureValuesAWS = func() []string {
```

Comment:
This is the same as `types.ArchitectureListAWS`, right? Can we keep this in a single place, probably best in this package?

Reply:
done.removed the `types.ArchitectureListAWS`